src/python/grpcio/grpc/_adapter/_types.py

```diff
@@ -321,6 +354,28 @@ def create_call(self, completion_queue, method, host, deadline=float('+inf')):
     """
     return None
 
+  @abc.abstractmethod
+  def connectivity_state(self, try_to_connect=False):
```

Comment:
Ah, I misunderstood. With respect to `grpc_channel_watch_connectivity_state`, I thought we discussed in a past meeting that we didn't want to use the queue?